src/utils/SConscript

```diff
@@ -4,16 +4,19 @@ import daos_build
 def scons():
     """Execute build"""
     Import('env', 'prereqs')
-
-    libs = ['daos', 'gurt', 'cart', 'daos_common', 'uuid']
+    libs = ['daos', 'gurt', 'cart', 'daos_common', 'uuid', 'dfs']
 
     denv = env.Clone()
+    denv.AppendUnique(LIBPATH=["../client/dfs"])
```

Comment:
hmm why are you adding this? dfs should be installed in ldfs and you add it two lines above.

Reply:
This is added in order to prevent a linker error
/bin/ld: cannot find -ldfs